public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx

```diff
@@ -231,11 +242,32 @@ const getStyles = (theme: GrafanaTheme2) => ({
       height: 'fit-content',
       borderRadius: theme.shape.radius.default,
     }),
+    container: css({
+      display: 'flex',
+      flexDirection: 'row',
+      padding: theme.spacing(1),
+      flex: 1,
+      width: '100%',
+    }),
     header: css({
       background: theme.colors.background.secondary,
       padding: `${theme.spacing(0.5)} ${theme.spacing(1)}`,
       borderBottom: `solid 1px ${theme.colors.border.weak}`,
       flex: 1,
     }),
+    button: css({
```

Comment:
I wonder if we can create a separate component from this style? IIRC we're using the same styles in the `features/expressions` components.

Reply:
I think it makes sense, as we want to keep consistency and not duplcate code. I will handle this in a separate PR 👍

Reply:
[PR](https://github.com/grafana/grafana/pull/94045) created!